linux_os/guide/system/permissions/partitions/mount_option_dev_shm_nosuid.rule

```diff
@@ -1,7 +1,5 @@
 documentation_complete: true
 
-prodtype: rhel6,rhel7,fedora,ol7
```

Comment:
Ubuntu:

```tmpfs on /dev/shm type tmpfs (rw,nosuid,nodev,size=503544k,nr_inodes=125886)```

Reply:
Looks like addable on OpenSUSE as they use it on `/tmp`:  https://forums.opensuse.org/showthread.php/519736-Cant-get-tmp-into-tmpfs